src/Pages/PopularRepositoriesTable/PopularRepositoriesTable.tsx

```diff
@@ -125,6 +136,7 @@ const PopularRepositoriesTable = () => {
   const storedPerPage = Number(localStorage.getItem(perPageKey)) || 20;
   const [page, setPage] = useState(1);
   const [searchValue, setSearchValue] = useState('');
+  const [searchQuery, setSearchQuery] = useState('');
```

Comment:
We already have a value that stores the search text, it is called "searchValue" 
This extra state is not needed.